packages/api/resolvers/types/product.js

```diff
@@ -1,9 +1,28 @@
 import { ProductTypes } from 'meteor/unchained:core-products';
 import { objectInvert } from 'meteor/unchained:utils';
+import crypto from 'crypto';
 
 export default {
   __resolveType(obj) {
     const invertedProductTypes = objectInvert(ProductTypes);
     return invertedProductTypes[obj.type];
   },
+  async catalogPrices(obj) {
```

Comment:
Didn't you move this function to the helpers lately?

Reply:
yes, I brought the logic from there. it's because I did know if there is any logic included for each type.